src/components/Connection.tsx

```diff
@@ -50,6 +50,10 @@ interface GridViewProps {
   isGridView: boolean;
   setIsGridView: React.Dispatch<React.SetStateAction<boolean>>;
 }
+interface DisplayProps {
+  isDisplay: boolean;
+  setIsDisplay: React.Dispatch<React.SetStateAction<boolean>>;
+}
```

Comment:
We don't need to define another interface if we are adding isDisplay and it's set variable in ConnectionProps, one interface is enough.

Reply:
okay I'll remove it.